### ISSUE TITLE
ASMP Amendment data support

### ISSUE THREAD

**Issue by alan-deanda** [cityofaustin/atd-data-tech#7523]
Provide support to the Systems Development team on the ASMP data. This issue covers the week starting on Monday Oct 25th. The team is expecting to publish a story map on Nov 1st.

**Comment by johnclary** [comment]
Waiting to hear from Cole.

**Comment by alan-deanda** [comment]
Currently helping troubleshoot field calculation errors and publishing story map themes.

**Comment by johnclary** [comment]
@alan-deanda would you add a summary here and close it out?

**Comment by alan-deanda** [comment]
The weeks leading up to publishing the apps consisted of configuring data, maps and apps to collect data in English and Spanish using the Crowdsource Polling template. Both apps had the same functionality. I translated the Spanish version myself. 
Once all crowdsource polling work was done, we integrated the apps into StoryMaps that were created by the SDD team. I reviewed and QA-ed the Spanish story map, and all assets were published.

The apps have been published for over a week now, and it seems that all usability issues that arose after deployment have been resolved.
